### ISSUE TITLE
Access for [Daniel Nalbach]

### ISSUE THREAD

**Issue by dnalbach** [department-of-veterans-affairs/va.gov-team#26020]
# Instructions
1. Fill out **Description**
2. Add the label used by your team (eg: `BAH-526`)
3. Do not remove `operations` label; this will notify the VSP Operations team of your request
4. When the issue is closed you will be notified and can continue with on-boarding setup

# Description
- Please provide your Contracting Officer's Representative (COR) name.
   > Mary O'Toole
- Please provide your name.
   > Daniel Nalbach
- Please Provide your email.
   > dnalbach@governmentcio.com
- Please provide the name of your team, your role on that team, and the name of the company you work for.
   > Team: CareGivers 
       Role: Senior Backend Engineer
       Company: GovernmentCIO

- Requesting SSH key authorization?
   > YES 
   - What is your **public** SSH key? If you don't have a SSH Key or know how to find it [click here](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/engineering/internal-tools.md#create-ssh-public-key) for instructions. (Example below)

```
ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABgQDOzjVD29RIoj+RcFvQAFD45rylOPmqYQl8UB/DaYn2JnS0wk7rUgb0wQFVAIRrT1BQOEOKT8ARHWRtR73VZbINrEzged/yPEQrMPbS7VXAF5GuV+LcEV+0BnjnAoieejDGVQ1lEw8KeYtLCghqIIc/uEnWPWhlgRN8pz5QOb3F99EfRuU64lUyUYnkDLkd6K0Cb/KWzAyWB5Rt3Duiwejb4duEgLwjbMn13+ZxZMYBIZwFtxNCMSTppZBTWOFhCwZQDkhN5/A1Sk2alR3we7KCCpbe1PaL/xF3uZTkGNwZO/5fe7slUfjQwKaUvR/6P+LyOVXc2a6nNNRU8Qv8+RBXtjz67GQ/G8BnBWmFz94InAwIudBg+D5AGMhKBSwpLdPQm2DgXZvp4mqKahCLv7i4xfSeTnTqVlpKOM4p1AGWFyJG0MRTPQdnlELLqmhfVDF5U3//gWl2sBDUjFQst29eew80Jk40ZYBSoCl+FZhMTPjFmZ62LmoD1C2nTn6/XdM= danielnalbach@dnalbach-C02FN16SML7J.local
```

- Grant AWS Console access?
  > YES
  - What AWS services do you need access to and why?
  - All AWS services. I will be supporting CareGivers, Healthcare, and also working on cross-team projects like reporting and analytics. I also request permissions to read, write, and execute Terraform resources. I have past AWS and DevOps experience.

**Comment by omgitsbillryan** [comment]
Waiting on @MaryO-USDS for approval. Email also sent:

![image](https://user-images.githubusercontent.com/3077884/121980103-624e1580-cd59-11eb-8746-d9bcec6993a3.png)

**Comment by MaryO-USDS** [comment]
Approved.

**Comment by omgitsbillryan** [comment]
I just became aware of the doc for AWS access and noticed that @MaryO-USDS is not on [the list of "Known Cors"](https://vfs.atlassian.net/wiki/spaces/VI/pages/1543143557/Granting+AWS+access#Known-CORs). Is this still good to go @dror?

Apologies in advance for being pedantic.

**Comment by Dror** [comment]
I keep getting notifications about different topics that I never participated in. 
So I just logged into this old user to tell you guys that you are mentioning the wrong Dror (:

**Comment by mchelen-gov** [comment]
@MaryO-USDS thanks for confirming!
@omgitsbillryan yup she is, updated doc!
@Dror loool thank you this is hilarious, people are mean @drorva and are off by 2 letters!

**Comment by mchelen-gov** [comment]
@dnalbach oops i am hearing that we need *your* contracting officer's name

**Comment by dnalbach** [comment]
Interesting @mchelen-gov , it looked like my peers all used Mary in their recent requests for access:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/24462
https://github.com/department-of-veterans-affairs/va.gov-team/issues/24334
https://github.com/department-of-veterans-affairs/va.gov-team/issues/24732
https://github.com/department-of-veterans-affairs/va.gov-team/issues/23869
https://github.com/department-of-veterans-affairs/va.gov-team/issues/21904

Mary also approved most everything else in my onboarding process forms and accesses. To clarify, are you needing a different 
 approver for AWS specifically? Any chance that would be Jeff Hayter? I notice he did extra approvals for two of my GCIO peers in requests above for AWS permissions:

https://github.com/department-of-veterans-affairs/va.gov-team/issues/24462
https://github.com/department-of-veterans-affairs/va.gov-team/issues/21904

If that doesn't sound right, I can ask around GCIO and see if anyone has ideas.

**Comment by dnalbach** [comment]
@Jeff-Hayter are you able to approve like you did for my coworkers? 
@mchelen-gov What is needed to further approve this request? Is it just Jeff?
---
FYI @mlighdc for your awareness

**Comment by mchelen-gov** [comment]
@dnalbach Sorry for the confusion, we're in the process of updating and clarifying this process. @omgitsbillryan please proceed with granting access

**Comment by drorva** [comment]
Since Mary approved above, approved.

**Comment by dnalbach** [comment]
Thank you all

**Comment by omgitsbillryan** [comment]
We don't currently give terraform access to folks not on the platform team. I've gone ahead and created a more typical VFS developer AWS account. If you require additional permissions for terraform, we should file a separate ticket for permissions escalation.
